meta-xt-control-domain-virtio/recipes-guest/domu/domu.bbappend

```diff
@@ -1,5 +1,10 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
+# virtio-env-weston.conf whould be used if we have the weston compositor in domd
+# the product should redefine variable.
+
+XT_DOMD_DISPLAY_SYSTEM ??= "weston"
```

Comment:
Oh, now you have this in two recipes... I believe it is better to move this variable definition to some common .inc file, to decrease code duplication.